### PR TITLE
android: Add verify contents buttons

### PR DESCRIPTION
**Pull request by t895**
Allows you to verify your game dumps and all installed content

game_content

installed_content

### CODE REVIEW

File: src/frontend_common/content_manager.h [hunk 1]
```diff
@@ -235,4 +243,129 @@ inline InstallResult InstallNCA(
...
+ * \param system Raw pointer to the system instance
+ * \param provider Raw pointer to the content provider that's tracking indexed games
+ * \param callback Optional callback to report the progress of the installation. The first size_t
+ * parameter is the total size of the installed contents and the second is the current progress. If
+ * you return true to the callback, it will cancel the installation as soon as possible.
+ * \return A list of entries that failed to install. Returns an empty vector if successful.
+ */
+inline std::vector<std::string> VerifyInstalledContents(
+    Core::System* system, FileSys::ManualContentProvider* provider,
+    const std::function<bool(size_t, size_t)>& callback = std::function<bool(size_t, size_t)>()) {
```
Comment (liamwhite) [inline src/frontend_common/content_manager.h:257]: Won't this break if the callback is not provided? (Shouldn't it always be provided for how we use it?)

Comment (t895) [reply src/frontend_common/content_manager.h:257]: It shouldn't be required but I should have wrote the default value with a return value

Comment (t895) [reply src/frontend_common/content_manager.h:257]: To be fair we never use it without a callback so I'll just remove the default value until we need something like that

File: src/android/app/src/main/jni/native.cpp [hunk 2]
```diff
@@ -829,6 +829,43 @@ void Java_org_yuzu_yuzu_1emu_NativeLibrary_removeMod(JNIEnv* env, jobject jobj,
                               program_id, GetJString(env, jname));
 }
 
+jobject Java_org_yuzu_yuzu_1emu_NativeLibrary_verifyInstalledContents(JNIEnv* env, jobject jobj,
+                                                                      jobject jcallback) {
+    auto jlambdaClass = env->GetObjectClass(jcallback);
```
Comment (FearlessTobi) [inline src/android/app/src/main/jni/native.cpp:834]: we use `snake_case` for C++ variable names

dito below

**Review by FearlessTobi — APPROVED** [review]

**Pull request merged**